### ISSUE TITLE
FileNotFoundError: [Errno 2] No such file or directory

### ISSUE THREAD

**Issue by satybald** [hcorona/WWC-recsys#12]
if you do fresh installation and try to download a files from notebook-1-data-gathering.
```
# downloads and unzips dataset from MovieLens 
#'http://files.grouplens.org/datasets/movielens/ml-latest-small.zip'
dataset_folder = os.getcwd()+'/data/'
dataset_folder_ready = utils.load_dataset(dataset_folder)
INFO:root:downloading dataset http://files.grouplens.org/datasets/movielens/ml-latest-small.zip
---------------------------------------------------------------------------
FileNotFoundError                         Traceback (most recent call last)
<ipython-input-4-fd4e2c2389df> in <module>()
      2 #'http://files.grouplens.org/datasets/movielens/ml-latest-small.zip'
      3 dataset_folder = os.getcwd()+'/data/'
----> 4 dataset_folder_ready = utils.load_dataset(dataset_folder)

~/Documents/programming/workshops/recsys-101-workshop/src/utils.py in load_dataset(datasets_folder, dataset_url)
     20     if not os.path.isfile(dataset_path):
     21         logging.info('downloading dataset %s', dataset_url)
---> 22         zf = urllib.request.urlretrieve(dataset_url, dataset_path)
     23         with zipfile.ZipFile(dataset_path, "r") as z: z.extractall(datasets_folder)
     24     else:

/usr/lib/python3.5/urllib/request.py in urlretrieve(url, filename, reporthook, data)
    196         # Handle temporary file setup.
    197         if filename:
--> 198             tfp = open(filename, 'wb')
    199         else:
    200             tfp = tempfile.NamedTemporaryFile(delete=False)

FileNotFoundError: [Errno 2] No such file or directory: '/home/ssatybaldiye/Documents/programming/workshops/recsys-101-workshop/data/ml-latest-small.zip'
```

**Comment by hcorona-zz** [comment]
I believe this is solved?

**Comment by satybald** [comment]
Yup, this issue is fixed via #11.
